clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -5280,9 +5280,13 @@ VarTemplateSpecializationDecl *Sema::BuildVarTemplateInstantiation(
 
   // TODO: Set LateAttrs and StartingScope ...
 
-  return cast_or_null<VarTemplateSpecializationDecl>(
+  auto *VD = cast_or_null<VarTemplateSpecializationDecl>(
       Instantiator.VisitVarTemplateSpecializationDecl(
           VarTemplate, FromVar, TemplateArgsInfo, Converted));
```

Comment:
@erichkeane, just a question here.  What happens to this VarTemplateSpecializationDecl case?  Does it just get dropped?

Reply:
VarTemplateSpecializationDecl inherits from VarDecl, so it 'just works' through the rest of the process if that is what you're asking?